### ISSUE TITLE
Uploading in folder shared via link not possible in Gallery view

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/gallery#475]
Use-case: I often share pictures with friends of an event or some travels we went to together. For that I share a folder with write permissions and send them the link.

Now for viewing, the Gallery view is much more awesome so when I send people a read-only link, the Gallery link is usually what I send them. However when I want them to upload their photos, I can’t do that because it’s not easily understandable how they can add their pictures.

TL;DR the shared link view of the Gallery app should also have an upload button in the breadcrumb, exactly like the Files app. That function should upload into the current folder which is being viewed.

cc @oparoz @owncloud/designers

**Comment by oparoz** [comment]
It's a duplicate of #225 ;)

**Comment by jancborchardt** [comment]
Ah, the ol’ self-duplicate. ;) Hopefully reinforces how needed that functionality is. :D
